src/main/java/jenkins/plugins/publish_over_ssh/BapSshHostConfiguration.java

```diff
@@ -79,14 +83,23 @@ public BapSshHostConfiguration() {
     @SuppressWarnings("PMD.ExcessiveParameterList") // DBC for you!
     @DataBoundConstructor
     public BapSshHostConfiguration(final String name, final String hostname, final String username, final String encryptedPassword,
-                                   final String remoteRootDir, final int port, final int timeout, final boolean overrideKey,
-                                   final String keyPath, final String key, final boolean disableExec) {
+                                   final String remoteRootDir, final String jumpHost, final int port, final int timeout, final boolean overrideKey, final String keyPath,
+                                   final String key, final boolean disableExec) {
         // CSON: ParameterNumberCheck
         super(name, hostname, username, null, remoteRootDir, port);
         this.timeout = timeout;
         this.overrideKey = overrideKey;
         this.keyInfo = new BapSshKeyInfo(encryptedPassword, key, keyPath);
         this.disableExec = disableExec;
+        this.jumpHost = jumpHost;
+    }
+
+    public String getJumpHost() {
+        return jumpHost;
+    }
+
+    public void setJumpHost(final String jumpHost) {
```

Comment:
If you add @DataBoundSetter above this, you do not need to add a parameter to the constructor.